src/utils/png.ts

```diff
@@ -58,7 +58,7 @@ export class PNG {
 
     this.rgbChannel = rgbChannel;
 
-    const hasAlphaValues = alphaChannel.some((a) => a < 1);
+    const hasAlphaValues = alphaChannel.some((a) => a < 255);
```

Comment:
Thanks for tracking this down and fixing it!!! Much appreciated. Would you mind adding a quick unit test to prevent regressions? This will require a new test file: `tests/utils/png.spec.ts`.

Reply:
@Hopding done! Let me know if it's okay (it's been a while since I wrote unit tests). That long line of Uin8Array might not be okay, but I'm not sure how to format that. It's ugly either way.. 😆 

The test itself passes with this PR (a < 255), and it fails without it (a < 1), so it's doing its job.

Reply:
Thanks @soadzoor. Just looked over it briefly and it looks good. I’ll try to get it merged this evening and released as soon as possible.